packages/support/resources/views/components/section/index.blade.php

```diff
@@ -124,6 +130,12 @@
                 </div>
             @endif
 
+            @if ($hasActions)
+                <div class="flex flex-1 justify-end">
```

Comment:
We can probably remove the wrapper and directly add a class like `ms-auto` to the actions component used here?

Reply:
thats better, done!

Reply:
the only difference I see is that now heading and action container is not taking half as no more `flex-1`
![image](https://github.com/filamentphp/filament/assets/67364036/7009a887-9ceb-4233-9301-46819ed3fdf2)

Reply:
Hmm but the heading wrapper has `flex-1`? 🤔

Reply:
yup it does